Hyperspace/Classes/Service/Network/NetworkActivityIndicatable.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  NetworkActivityIndicatable.swift
+//  Hyperspace-iOS
+//
+//  Created by William McGinty on 12/21/17.
+//  Copyright © 2017 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+
+/// Represents an object that can indicate to the user that network activity is currently taking place.
+public protocol NetworkActivityIndicatable {
+    var isNetworkActivityIndicatorVisible: Bool { get set }
+}
+
+/// Represents an object that is capable of recording the state of network requests and forwarding that state onto the corresponding indicator
```

Comment:
Small documentation tweak ("onto" --> "to):
`/// Represents an object that is capable of recording the state of network requests and forwarding that state to the corresponding indicator`